airflow/providers/mysql/operators/mysql.py

```diff
@@ -42,7 +42,7 @@ class MySqlOperator(SQLExecuteQueryOperator):
 
     .. seealso::
         For more information on how to use this operator, take a look at the guide:
-        :ref:`howto/operator:MySqlOperator`
+        :ref:`howto/operator:mysql`
```

Comment:
Do we really need to change target from `MySqlOperator` to generic `mysql` ?

Reply:
Yes. The doc checks are failing as this reference is not available.